intro_to_rl.ipynb

```diff
@@ -0,0 +1,3225 @@
+{
```

Comment:
<p>Nit: full name is used for all variables except <code>obs</code> . I think it will be more understandable to make this into observation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>